### ISSUE TITLE
http://wpad/wpad.dat

### ISSUE THREAD

**Issue by perrog** [dotnet/docs#18231]
Step 4 could mention try visit http://wpad/wpad.dat to load the configuration file, so readers experience  Eureka how auto-proxys work. ”DNS is queried for a host with WPAD as its name or alias” is functionally correct but summary the full WDAP protocol in one single sentence,which makes it hard to understand.

This legacy MSDN article is very useful:
https://docs.microsoft.com/en-us/archive/msdn-magazine/2005/august/take-the-burden-off-users-with-automatic-configuration-in-net

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a66edea4-6660-f670-fe9d-94302ac98cb3
* Version Independent ID: 24143fa6-d272-f5d6-eef4-c05f6497d99e
* Content: [Automatic Proxy Detection](https://docs.microsoft.com/en-us/dotnet/framework/network-programming/automatic-proxy-detection)
* Content Source: [docs/framework/network-programming/automatic-proxy-detection.md](https://github.com/dotnet/docs/blob/master/docs/framework/network-programming/automatic-proxy-detection.md)
* Product: **dotnet-framework**
* Technology: **dotnet-networking**
* GitHub Login: @karelz
* Microsoft Alias: **ncldev**

---
[Associated WorkItem - 558832](https://dev.azure.com/msft-skilling/Content/_workitems/edit/558832)

**Comment by karelz** [comment]
@perrog do you mean adding something like "(looking for file http://wpad/wpad.dat)" in step [3]? (not 4 as you mentioned)
I am not familiar with WDAP protocol, but if there is confirmation somewhere, this is the URL that is always searched for (and there are no alternatives/configurations), I wouldn't mind mentioning it here.
Going into depth of WDAP protocol is IMO beyond the scope of this article.

cc @davidsh

**Comment by davidsh** [comment]
This is the best documentation for WPAD protocol, PAC files, etc.
http://findproxyforurl.com/

**Comment by karelz** [comment]
Is it something you would recommend to link from the article? Or maybe apply some of the suggestions above @davidsh?

**Comment by davidsh** [comment]
>Is it something you would recommend to link from the article? Or maybe apply some of the suggestions above @davidsh?

Yes, probably a mixture of both ideas. Summarize some of the important points but add a link to this website since we don't want to replicate all of the information in our docs itself.
